### PR TITLE
fix(style):

### PR DESCRIPTION
**Pull request by devopsmariocom**
- title
- logo
- custom color scheme
- replace Lorem ipsum with TBA
- fix sponsor size and caption

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -172,11 +171,11 @@ <h3 class="md-display-1">{{ ::$ctrl.data.sections.sponsors.title }}</h3>
       <p ng-repeat="content in ::$ctrl.data.sections.sponsors.content">
         {{ ::content }}
       </p>
-      <div layout-xs="column" layout="row" layout-align="center center" class="sponsors">
+      <div layout-xs="column" layout="column" layout-align="center center" class="sponsors">
```
Comment (Hotell) [inline src/app/app.component.html]: this is reduntant, if you wanna have always column flex layout just remove `layout-xs`

File: src/app/app.component.html [hunk 2]
```diff
@@ -184,7 +183,6 @@ <h3 class="md-display-1">{{ ::$ctrl.data.sections.sponsors.title }}</h3>
         <p ng-repeat="footer in ::$ctrl.data.sections.sponsors.footer">
           {{ ::footer }}
         </p>
-        <md-button ng-href="{{ ::$ctrl.data.ngParty.links.social.meetup }}" class="md-accent" flex-xs="">get in touch</md-button>
```
Comment (Hotell) [inline src/app/app.component.html:187]: why did you remove get in touch btn? I mean it's up to discussion

File: src/app/app.component.html [hunk 3]
```diff
@@ -221,7 +219,7 @@ <h3 class="md-display-1">{{ ::$ctrl.data.sections.codeOfConduct.title }}</h3>
         </li>
       </ul>
 
-      <small>&copy; ngPartyCz 2016</small>
+      <small>&copy; ngPartyCz 2016 made with love and ng-metadata</small>
```
Comment (Hotell) [inline src/app/app.component.html:225]: haha I like this advert! :dancers:

**Comment by Hotell** [comment]
Overall  good job! Just tweak those things in comment and we are good to go. thx!

File: src/app/app.component.html (hunk 2 above)
Comment (devopsmariocom) [inline src/app/app.component.html:187]: moved to the bottom

**Comment by Hotell** [comment]
Gracias amigo!
